hack/second-controller.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python3
+# Author: Chmouel Boudjnah <chmouel@redhat.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import os
+import sys
+
+import argparse
+import yaml
+
+
+HELP_DESC = """This script will add a second controller to pipelines-as-code
+
+It's designed to be used like this:
+
+kubectl apply -f <(python3 yamlconvert.py LABEL)
+
+If you want to build the controller with ko you can use the
+--controller-image=ko, this will (assuming you have a KO_DOCKER_REPO env
+variable set properly) use ko to deploy the second controller image in the
+target namespace (default: pipelines-as-code):
+
+ko apply -f <(python3 yamlconvert.py --controller-image=ko LABEL)
+
+You can define a few env variables to change the behavior:
+
+• PAC_CONTROLLER_LABEL: the label of the controller
+• PAC_CONTROLLER_TARGET_NS: the namespace to deploy the controller to (default:
+                            pipelines-as-code)
+• PAC_CONTROLLER_SECRET: the name of the secret to use (default: LABEL-secret)
+• PAC_CONTROLLER_CONFIGMAP: the name of the configmap to use (default: LABEL-configmap)
+• PAC_CONTROLLER_SMEE_URL: the url to use if you want to deploy a gosmee
+                           container to this controller. If not set, it won't deploy it.
+• PAC_CONTROLLER_IMAGE: the image name of the controller. use the word "ko" to use ko to build the image
+                       (default: ghcr.io/openshift-pipelines/pipelines-as-code-controller:stable)
+"""
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(
+        description="Generate yaml to add second paac controller",
+        epilog=HELP_DESC,
+        formatter_class=argparse.RawTextHelpFormatter,
+    )
+    parser.add_argument(
+        "label",
```

Comment:
```suggestion
        "--label",
``` 
to be consistent with other?

Reply:
ah I see its not required 👍

Reply:
yeah :D dashes `--` are for non required stuff and just a arg make it required, and since we need a label for all things then it's an argument